src/main/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/messages.properties

```diff
@@ -20,6 +20,7 @@ javadoc.noPeriod=First sentence should end with a period.
 javadoc.packageInfo=Missing package-info.java file.
 javadoc.paragraph.line.before=<p> tag should be preceded with an empty line.
 javadoc.paragraph.misplaced.tag=<p> tag should be placed immediately before the first word, with no space after.
+javadoc.paragraph.preceded.block.tag=<p> tag should not precede HTML block-tag ''<{0}>'', it should be removed.
```

Comment:
`<p> tag should not precede HTML block-tag ''<{0}>'', <p> should be removed.`

to avoid any possible confusion what should be removed.

Reply:
fixed

Reply:
also updated translations